src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -460,11 +460,19 @@ private static bool CanSkipDocument(Document document, BuildVersionType buildTyp
                 return false;
 
             // skipping "Raven/Replication/DatabaseIdsCache" and
-            // "Raven/Replication/Sources/{GUID}"
-            if (document.Id.Size != 34 && document.Id.Size != 62)
+            // "Raven/Replication/Sources/{GUID}" and
```

Comment:
What is this? I think it is dead. 
In general, all of these document id aren't valid any longer.